src/nix/flake.cc

```diff
@@ -1144,6 +1145,11 @@ struct CmdFlakeShow : FlakeCommand, MixJSON
             .description = "Show the contents of outputs for all systems.",
             .handler = {&showAllSystems, true}
         });
+        addFlag({
+            .longName = "allow-import-from-derivation",
```

Comment:
Mhm, can it be somehow an issue to collide with the existing? `Setting<bool> enableImportFromDerivation`, because all options are also flags...

Reply:
I'm not a 100% sure on that, but in retrospect reusing the same flag `nix` uses for `nix flake show` isn't ideal. I'll do a `s/allow/evaluate/` to dodge the collusion and make the intent clearer